can/io/blf.py

```diff
@@ -87,7 +87,12 @@ def __init__(self, filename):
         header = FILE_HEADER_STRUCT.unpack(data)
         #print(header)
         assert header[0] == b"LOGG", "Unknown file format"
+        self.objectCount = header[12]
         self.start_timestamp = systemtime_to_timestamp(header[14:22])
+        self.stop_timestamp = systemtime_to_timestamp(header[22:30])
+
+    def __len__(self):
+        return self.objectCount
```

Comment:
This is not really accurate as object count includes all objects while `__len__()` should return number of CAN messages + error frames as that is what you get when you iterate over this object.